mobify/mobifyjs
# Mobifyjs takedown

Status: **Ready for Review**
Owner: @ericawright
Reviewers: @stewartyu @johnboxall 
## Changes
- redo mobifysj homepage to have a message that we are no longer supporting it
- redirect related pages to /mobifyjs
## Todos:
### Feedback:

_none so far_
## How to Test
- `grunt jekyll`
- observe http://localhost:4000/mobifyjs/ (should not redirect)
- observe http://localhost:4000/mobifyjs/v2/docs (should redirect)
